options/ts.ts

```diff
@@ -15,6 +15,7 @@ function getTsTaskOptions(grunt: IGrunt, tsconfig: any): any {
 	const distCompilerOptionsOverride = tsOverrides && tsOverrides.dist && tsOverrides.dist.compilerOptions ? tsOverrides.dist.compilerOptions : {};
 	tsconfigDist.compilerOptions = Object.assign({}, tsconfig.compilerOptions, {
 		outDir: distDir,
+		mapRoot: '../dist/umd/_debug',
```

Comment:
A bigger question I support, I was effectively reverting back to the previous configuration that had regressed.